src/Microsoft.Data.SqlClient/netcore/src/Resources/Strings.Designer.cs

```diff
@@ -889,7 +889,7 @@ internal class Strings {
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to The validation of an attestation token failed. The token signature does not match the signature computed using a public key retrieved from the attestation public key endpoint at &apos;{0}&apos;. Verify the DNS mapping for the endpoint. If correct, contact Customer Support Services..
+        ///   Looks up a localized string similar to The validation of an attestation token failed. The token signature does not match the signature computed using a public key retrieved from the attestation public key endpoint at &apos;{0}&apos;. Verify the DNS mapping for the endpoint - see https://go.microsoft.com/fwlink/?linkid=2157649 for more details. If correct, contact Customer Support Services..
```

Comment:
Since you are touching this file, can we fix the double dots at the end of these sentences?

Reply:
nvm, there are too many of them. Just leave them as they are before. No I see they are just because of the long embedded sentences.

Reply:
It's auto-generated file.